client/package.json

```diff
@@ -3,11 +3,14 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@types/jquery": "^3.3.31",
     "axios": "^0.19.0",
     "bootstrap": "^4.3.1",
     "dotenv": "^8.0.0",
     "font-awesome": "^4.7.0",
     "history": "^4.9.0",
+    "jquery": "^3.4.1",
```

Comment:
jquery?

Reply:
I will delete it. It is not needed anymore